src/components/Wizard/useFormWizard.ts

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from "react";
+
+export const useFormWizard = ({
+  steps,
+  complete,
+}: {
+  steps: string[];
+  complete: () => void;
+}) => {
+  const [history, setHistory] = useState<string[]>([]);
+  const [currentStep, setCurrentStep] = useState<number>(0);
+  useEffect(() => {
+    if (currentStep === steps.length) {
+      complete();
+    }
+  }, [currentStep]);
+  const goToPrevious = () => {
```

Comment:
Do you want to do any bounds checking on this function and `goToNext`?

Reply:
Updated!